quickstarts/playwright-check.mdx

```diff
@@ -63,80 +70,95 @@ Detailed session summary at: https://chkly.link/l/linkhere
 ```
 
 
-### 4. Convert tests to checks with `pw-test`:
+### 4. Convert tests to monitoring checks with `pw-test --create-check`:
 
-Using `pw-test` with the `--create-check` flag will create a `checkly.config.ts` file if it doesn't exist, and will add the new check. Afterwards, you can tweak the monitoring configuration for your check.
+The `pw-test` command enables you to run your existing tests in the Checkly infrastructure. Use the `--create-check` flag to create a `checkly.config.ts` file if it doesn't exist. The `checkly.config` will include the necessary configuration to run your Playwright tests as synthetic monitors. and will add the new check.
 
 For example:
 
 ```bash
-> npx checkly pw-test --create-check -- --project=chromium
+$ npx checkly pw-test --create-check -- --project=chromium
 ⚠ No Checkly config file found
 
 ℹ Creating a default checkly config file.
 
 Creating Checkly check from Playwright test... ✅
 ```
 
-Review the resulting check configuration in your `checkly.config.ts` file and make sure to tweak locations and schedule as needed.
+Review the resulting check configuration in your `checkly.config.ts` file and make sure to adjust locations and schedule as needed.
 
-```typescript {title="checkly.config.ts"}
+```typescript {title="checkly.config.ts"} highlight={8-18}
 import { defineConfig } from 'checkly'
 
 const config = defineConfig({
-  logicalId: 'my-repo-name',
-  projectName: 'my-repo-name',
+  logicalId: 'playwright-check-suites',
+  projectName: 'my-repo-playwright-check-suites',
   checks: {
     playwrightConfigPath: './playwright.config.ts',
     playwrightChecks: [
       {
-        logicalId: 'playwright-check-project-chromium', // tweak ID
-        name: 'Playwright Test: --project=chromium', // tweak name
-        testCommand: 'npx playwright test --project=chromium', // what we'll run as part of your check suite
+        logicalId: 'playwright-check-project-chromium', // Playwright Check Suite id
+        name: 'Playwright Test: --project=chromium', // Playwright Check Suite name
+        testCommand: 'npx playwright test --project=chromium', // Command to run your tests
```

Comment:
shall we add installCommand in here to educate on it? If they want less or more dependencies, this is how they rule over them. 
```suggestion
        installCommand: 'npm install --dev' // Command to install your  code dependencies
        testCommand: 'npx playwright test --project=chromium', // Command to run your tests
```